lib/AckGroupingTrackerEnabled.h

```diff
@@ -71,6 +71,15 @@ class AckGroupingTrackerEnabled : public AckGroupingTracker {
     //! Method for scheduling grouping timer.
     void scheduleTimer();
 
+    //! State
+    enum State
+    {
+        NotStarted,
+        Ready,
+        Closed,
+    };
+    std::atomic<State> state_;
```

Comment:
The `NotStarted` and `Closed` states are never used, would it better to just use a `atomic_bool` here?